### ISSUE TITLE
ReadTheDocs font is underspecified

### ISSUE THREAD

**Issue by clinssen** [nest/nest-simulator#2322]
Opening https://nest-simulator.readthedocs.org/ in Chrome 91 on Debian Linux renders with a fixed-width font.

It seems that the computed font for the main text is

```css
system-ui, -apple-system, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", "Liberation Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji"
```

I fear that "system-ui" is not specific enough to achieve a consistent rendering across platforms.

**Comment by jessica-mitchell** [comment]
@clinssen there is indeed some odd behaviour with the font. I explicitly added a font-family to the custom css. Could you check on your system how this looks: https://nest-test.readthedocs.io/en/fix-font-rtd/index.html  
thanks!

Also, fyi, the font-family in for the forthcoming new theme (#2289), is included in the css as well.

**Comment by jessica-mitchell** [comment]
Marking this as closed since #2289 should have resolved font issues. Please create a new issue if issue comes up again
